core/lib/multivm/src/versions/vm_fast/version.rs

```diff
@@ -21,7 +21,8 @@ impl TryFrom<VmVersion> for FastVmVersion {
     fn try_from(value: VmVersion) -> Result<Self, Self::Error> {
         match value {
             VmVersion::Vm1_5_0IncreasedBootloaderMemory => Ok(Self::IncreasedBootloaderMemory),
-            VmVersion::VmGateway => Ok(Self::Gateway),
+            // FIXME: implement differentiated memory model in fast VM
+            VmVersion::VmGateway | VmVersion::VmEvmEmulator => Ok(Self::Gateway),
```

Comment:
is it tracked by someone?

Reply:
@joonazan @slowli

Reply:
There is https://github.com/matter-labs/vm2/pull/74, which should be extended to switch between memory models on demand. cc @joonazan